protobuf_to_dict/convertor.py

```diff
@@ -192,25 +195,33 @@ def _get_field_mapping(pb, dict_value, strict):
     return field_mapping
 
 
-def _dict_to_protobuf(pb, value, type_callable_map, strict, ignore_none):
+def _dict_to_protobuf(pb, value, type_callable_map, strict, ignore_none, ignore_values):
     fields = _get_field_mapping(pb, value, strict)
 
     for field, input_value, pb_value in fields:
-        if ignore_none and input_value is None:
+        if ignore_none and input_value is None:  # To deprecate. Use `ignore_values={None}` instead.
             continue
+        if ignore_values:
+            try:
```

Comment:
As you said, it looks kind of ugly to have the try except here.
What do you think about:

`if getattr(input_value, '__hash__') and input_value in ignore_values`,
then you won't need the try except clause.